### DISCUSSION TITLE
How to make only placeholder italics in tailwind CSS ?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#201] [Q&A] [answered]
In short.

**Comment by d4tdev** [comment, marked as answer]
> 1. You can use like this:

 class="placeholder:italic"

> 2. Define this

@layer utilities {
  .placeholder-italic::placeholder{
    @apply italic
  }
}
> And use  

class="placeholder-italic"
